src/hotspot/share/gc/shenandoah/heuristics/shenandoahOldHeuristics.hpp

```diff
@@ -145,7 +145,8 @@ class ShenandoahOldHeuristics : public ShenandoahHeuristics {
   // last_old_region_index() entries, or memory may be corrupted when this function overwrites the
   // end of the array.
   unsigned int get_coalesce_and_fill_candidates(ShenandoahHeapRegion** buffer);
-  bool has_coalesce_and_fill_candidates() { return _last_old_region > 0; }
+  bool has_coalesce_and_fill_candidates() const { return _last_old_region > 0; }
+  size_t coalesce_and_fill_candidates() const { return _last_old_region; }
```

Comment:
The return value from `coalesce_and_fill_candidates()` (which by the way could be confused for an imperative action verb) lready available in `last_old_region_index()` although might be a good idea to rename appropriately? Also fine to leave as is, but it sounds to me like one name might work for both uses.

Reply:
I agree, one (good) name for both is best.